.php-cs-fixer.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+return Ibexa\CodeStyle\PhpCsFixer\InternalConfigFactory::build()
```

Comment:
```suggestion
return Ibexa\CodeStyle\PhpCsFixer\Config::create()
```

This is community package so `\Ibexa\CodeStyle\PhpCsFixer\Config` should be used.